lizard.py

```diff
@@ -759,8 +766,14 @@ def get_warnings(code_infos, option):
 
 def print_result(result, option, scheme):
     result = print_and_save_modules(result, option.extensions, scheme)
+    warning_count = 0
+    warning_nloc = 0
     warnings = get_warnings(result, option)
-    warning_count, warning_nloc = print_warnings(option, scheme, warnings)
+    if list(warnings):
```

Comment:
`get_warnings()` returns a generator so `if warnings:` will never be false. So to check that the generator is empty (meaning no warnings) I use `list(warnings)`